src/main/java/org/entando/entando/aps/system/init/model/ComponentInstallationReport.java

```diff
@@ -36,11 +36,15 @@ protected ComponentInstallationReport(Element element) {
 		Element dataElement = element.getChild(SystemInstallationReport.DATA_ELEMENT);
 		if (null != dataElement) {
 			this.setDataReport(new DataInstallationReport(dataElement));
-		}
+		} else {
```

Comment:
Is there some problem with the formatting?

Reply:
Fixed